contents/blog/how-we-improved-feature-flags-resiliency.md

```diff
@@ -0,0 +1,145 @@
+---
+date: 2023-03-28
+title: How we improved feature flag resiliency
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+---
+
+Feature flags as a service is an interesting space to be working in. If your service stops working, not only does it affect your customers, but also your
+customer's customers, since they rely on you to make sure their app works fine.
```

Comment:
```suggestion
Feature flags as a service is an interesting space to be working in. If your service stops working, not only does it affect your customers, but also your customer's customers, since they rely on you to make sure their app works.
```